modules/firewall/outputs.tf

```
@@ -1,4 +1,4 @@
 output "network_settings_id" {
     description = "ID of the GitHub.Network/networkSettings resource"
-    value = jsondecode(azurerm_resource_group_template_deployment.github_network_settings.output_content).gitHubId.value
+    value = jsondecode(azapi_resource.github_network_settings.output).gitHubId.value
```

This is wrong and results in a runtime error, it should be:
```suggestion
    value = jsondecode(azapi_resource.github_network_settings.output).tags.GitHubId
```